cmd/zpool/zpool_vdev.c

```diff
@@ -212,9 +212,9 @@ static vdev_disk_db_entry_t vdev_disk_database[] = {
 #if 0
 	{"ATA     QEMU HARDDISK   ", 8192},
 	{"IET     VIRTUAL-DISK    ", 8192},
-	{"OI      COMSTAR         ", 8192},
-	{"SUN     COMSTAR         ", 8192},
-	{"NETAPP  LUN             ", 8192},
+	{"OI      COMSTAR	 ", 8192},
```

Comment:
This looks wrong. If the GitHub reviewer is showing this correctly, you're improperly replacing spaces with tabs here.